frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -472,7 +510,22 @@ const SelectDateTime = ({
           </FormControl>
         )}
       {isBeingEdited ? (
-        <SaveButton onSaveClick={onSaveClick} />
+        <>
+          {formValues.recurringDonationId !== "null" ? (
```

Comment:
null here or ""?

Reply:
It is always either "null" for one time donation or a number for recurring. It is never "".